src/app/codegen-data-model-provider/CodegenDataModelProvider.cpp

```diff
@@ -14,6 +14,7 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+#include "app/util/att-storage.h"
```

Comment:
Why is this include needed?

Reply:
This file uses CLUSTER_MASK_CLIENT which is included in this header file.

Reply:
Removed